packages/MarkdownEditor-Core.package/MarkdownDelimiterEmphasis.class/instance/attributeStartingAt..st

```diff
@@ -0,0 +1,10 @@
+attributes
+attributeStartingAt: openingMarkdownDelimiterEmphasis
+
+	| attributeDelimiterLength start end attributeClass |
+
+	attributeDelimiterLength := (self attributeDelimiterLengthWith: openingMarkdownDelimiterEmphasis).
+	start := openingMarkdownDelimiterEmphasis endPosition - attributeDelimiterLength + 1.
+	end := self startPosition + attributeDelimiterLength - 1.
+	attributeClass := self attributeTypeStartingAt: openingMarkdownDelimiterEmphasis.
```

Comment:
Isn't the type dependant on the `attributeDelimiterLength`? The method body of `attributeTypeStartingAt:` could be simpler when using only the length imo

Reply:
Do I understand correctly that you want the method `attributeTypeStartingAt:` to call `attributeDelimiterLengthWith:`?

Reply:
No, I thought of a method which maps `1` to italic and `2` to bold (which is more explicit how you are doing the mapping). The variable `attributeDelimiterLength` already contains the length, so why not use it?